tests/e2e/standalone/standalone_test.go

```diff
@@ -477,17 +477,14 @@ func testRunApiLogLevel(t *testing.T) {
 
 	args = []string{
 		"run",
-		"--app-id", "apiloglevel_debug",
-		"--api-log-level", "debug",
-		"--log-level", "debug",
+		"--app-id", "enableApiLogging_info",
 		"--", "bash", "-c", "echo 'test'",
 	}
 
-	t.Run(fmt.Sprintf("check apiloglevel flag debug mode"), func(t *testing.T) {
+	t.Run(fmt.Sprintf("check enableApiLogging flag in disabled mode"), func(t *testing.T) {
 		output, err := spawn.Command(daprPath, args...)
 		t.Log(output)
 		require.NoError(t, err, "run failed")
-		assert.Contains(t, output, "level=debug msg=\"HTTP API Called: PUT /v1.0/metadata/appCommand\"")
```

Comment:
do not remove this assert.
Make sure that line is not there in the output. 
Do a negative assertion.